distributions/Dist_yl4874.py

```diff
@@ -0,0 +1,24 @@
+#Yongning Lei (Intro to Experimental Phy Lab II) P = lnx
+
+import numpy as np
+from .base_distribution import BaseDistribution
+
+
+class Dist_yl4874(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = 0
+		self.x_max = 1
+
+
+	def pdf(self, x):
+
+		return np.abs(lnx) #normalizing lnx  in bounds 0,1 gives us the function abs(lnx)
```

Comment:
this is not valid python syntax. what is `lnx`

Reply:
oh that's my fault, so what i did is import math and using np.abs(math.log（x）) instead